src/frontends/pytorch/src/op/expm1.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/subtract.hpp"
+#include "openvino/opsets/opset7.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+
+OutputVector translate_expm1(const NodeContext& context) {
+    num_inputs_check(context, 1, false);
```

Comment:
I think it is not the correct value for the third argument, could you please check this function code?
https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/pytorch/src/utils.cpp#L20

False probably will be treated as 0 for max_inputs argument and this check never passed as your operation always has 1 input and 1 > 0